src/content/guides/mml-collide-events-guide/index.ts

```diff
@@ -0,0 +1,18 @@
+import { DocsGuide } from "@/types/docs-guides";
+
+import body from "./body.mdx";
+
+const collisionEvents: DocsGuide = {
+  title: "Getting Started: Collision Events in MML",
+  description: "Learn how to leverage the power of the collisions.",
+  body,
+  linkList: [
+    "Related events",
+    "Related attributes",
+    "Basic example of element collision",
+    "Example with multiple users and disconnection events",
+    "Advanced Example with collisionmove events",
```

Comment:
```suggestion
    "Advanced example with collisionmove events",
```

(to be consistent with others)